DSCResources/MSFT_xWebsite/MSFT_xWebsite.psm1

```diff
@@ -323,24 +332,26 @@ function Set-TargetResource
                 $ErrorMessage += $_.Exception.Message
                 $Exception = New-Object -TypeName System.InvalidOperationException -ArgumentList $ErrorMessage
                 $ErrorRecord = New-Object -TypeName System.Management.Automation.ErrorRecord -ArgumentList $Exception, $ErrorId, $ErrorCategory, $null
+
                 $PSCmdlet.ThrowTerminatingError($ErrorRecord)
             }
         }
     }
-    else #Ensure is set to "Absent" so remove website
+    else # Remove website if Ensure is set to 'Absent'
     {
         try
         {
             $Website = Get-Website | Where-Object -FilterScript {$_.Name -eq $Name}
 
-            if ($Website -ne $null)
+            if ($Website)
```

Comment:
Not sure I really understand the point of this check in the first place.

To get here we'd have to get a false on test-targetresource when ensure = 'absent' and it already find a site. Why are we bothering to check again?